static/json/languages/en_US.json

```diff
@@ -1306,6 +1306,8 @@
   "nfts": {
     "empty_state_header": "NFTs",
     "empty_state_description": "Your NFTs will appear here!",
+    "report_nft_action_text": "Report",
+    "report_nft_confirm_description": "Confirm report of NFT?",
```

Comment:
```suggestion
    "report_nft_confirm_description": "Are you sure you want to report this NFT?",
```

Reply:
Tried that approach, but seems like the text break point doesn't seem too great. "Are you sure you want to report this" is in one line and then the next line you have "NFT?".